interview/infra/tts/polly_client.py

```
@@ -0,0 +1,40 @@
+import os
+import uuid
+import boto3
+
+class PollyClient:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)